src/views/Account/AccountBalances.js

```diff
@@ -236,98 +72,136 @@ class AccountBalances extends Component {
-              <StyledAsset data-toggle="tooltip" title={token.name}>
+              <StyledAsset 
+                onMouseEnter={() => this.onAssetHover(token.address)}
+                onMouseLeave={() => this.onAssetHover(null)}
+              >
                 <AssetIcon asset={token.address} />
-                <p>{ellipseText(token.name, 30)}</p>
+                <p>{token.name}</p>
+                {this.state.hoveredAsset === token.address && 
+                  <StyledShowIcon onClick={() => this.onShowAsset(token.address)}/>}
               </StyledAsset>
               <p>{token.balance.display}</p>
-              <p>{token.native ? token.native.price.display : '———'}</p>
+              <p>{token.native ? token.native.price.display : emptyBalanceDisplay}</p>
               <StyledPercentage
                 percentage={
                   token.native
                     ? convertStringToNumber(token.native.change.amount)
                     : 0
                 }
               >
-                {token.native ? token.native.change.display : '———'}
+                {token.native ? token.native.change.display : emptyBalanceDisplay}
               </StyledPercentage>
-              <p>{token.native ? token.native.balance.display : '———'}</p>
+              <p>{token.native ? token.native.balance.display : emptyBalanceDisplay}</p>
             </StyledToken>
-          ))}
-        <StyledLastRow>
-          {!!tokensToggleDisplay.length && !this.state.disableToggle ? (
-            <StyledShowMoreTokens onClick={this.onShowMoreTokens}>
-              <ToggleIndicator show={this.state.showMoreTokens} />
-              {`${
-                this.state.showMoreTokens
-                  ? lang.t('account.hide')
-                  : lang.t('account.show')
-              } ${tokensToggleDisplay.length} ${
-                tokensToggleDisplay.length === 1
-                  ? lang.t('account.token')
-                  : lang.t('account.tokens')
-              } ${
-                allLowMarketTokensHaveNoValue
-                  ? lang.t('account.no_market_value')
-                  : lang.t('account.low_market_value')
-              }`}
-            </StyledShowMoreTokens>
-          ) : (
-            <div />
-          )}
-          <p>{`${this.props.accountInfo.total.display || '———'}`}</p>
-        </StyledLastRow>
+          ))
+        }
       </StyledGrid>
     );
   }
+  _getAssets() {
+    const tokens = this.props.accountInfo.assets.filter(
+      asset => asset.symbol !== 'ETH' && typeof asset === 'object' && !!asset,
+    );
+
+    const tokensWithNoMarketValue = tokens.filter(asset => !asset.native);
+    const tokensWithLowMarketValue = tokens.filter(
+        asset => asset.symbol !== 'ETH' && (hasLowMarketValue(asset))
+    );
+    const dustTokens = [
+      ...tokensWithLowMarketValue,
+      ...tokensWithNoMarketValue,
+    ];
+
+    //const hiddenTokens = [...dustTokens.map(t => t.address), ...this.props.hiddenAssets];
```

Comment:
Still not sure if we should automatically mark tokens that have low/no value ourselves or just leave to the user. If we do it automatically we need to keep track of two lists. I think I'll go with this: 
- If the local storage doesn't have any hiddenAssets, create the array and put in all shitcoins with low value
- After the local storage has an array just leave it up to the user